### ISSUE TITLE
Turn off learning rate scheduler

### ISSUE THREAD

**Issue by 01bui** [ultralytics/ultralytics#1463]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi,

I want to turn off the learning rate scheduler in yolov8. Does anyone know how to do this?

Thanks.

**Comment by glenn-jocher** [comment]
@01bui you can adjust the LR hyps used to train. If you set lrf=1.0 then you are "turning off" the scheduler as you say. See https://docs.ultralytics.com/modes/train/#arguments

**Comment by 01bui** [comment]
> @01bui you can adjust the LR hyps used to train. If you set lrf=1.0 then you are "turning off" the scheduler as you say. See https://docs.ultralytics.com/modes/train/#arguments

Thanks. How do I add the learning rate over epochs to Tensorboard? I only see metrics and loss on tensorboard for now.

**Comment by glenn-jocher** [comment]
@01bui, to add the learning rate over epochs to Tensorboard, you first need to define a `Tensorboard` object in the `train.py` script using `SummaryWriter`. Then you can log the learning rate value as follows:

```
writer.add_scalar('LR/lr', scheduler.get_last_lr()[0], global_step)
```

This logs the current learning rate value with the tag `lr` under the `LR` directory in Tensorboard. Please note that `scheduler.get_last_lr()[0]` returns the current learning rate value. 

You can then visualize the changes in learning rate over epochs in Tensorboard using the `lr` tag.
